src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/flyout/__jest__/flyout.test.js

```diff
@@ -120,6 +120,21 @@ describe('Flyout', () => {
     expect(component.state('file')).toBe(mockFile);
   });
 
+  it('should allow removing a file', async () => {
+    const component = shallowWithIntl(<Flyout.WrappedComponent {...defaultProps} />);
+
+    // Ensure all promises resolve
+    await new Promise(resolve => process.nextTick(resolve));
```

Comment:
I may be mistaken, but I *think* `Promise.resolve` would resolve async:

```js
await Promise.resolve();
```

Reply:
You are right, I like it. I made the change and will merge after CI is green. Thanks for the reivew 👍